### PR TITLE
AGORA-1366/Port the HoverCard when hovering over a delegate on a vote table view

### PR DESCRIPTION
**Pull request by wojtekwtf**
This PR adds a HoverCard in the vote table view

Updated video (V3): https://www.loom.com/share/1ca36c1a70f44120905ddc4e9394f9fa

✅ Functionality matches preview version, includes the voting power, delegate button, and social media buttons

❓ It loads very slowly on my computer. `fetchDelegate` and `fetchBalanceForDirectDelegation` are taking a lot of time to display the delegate information and then to display the delegate button. Should i optimize it?

### CODE REVIEW

**Comment by yitongzhang** [comment]
> ❓ Shadcn default view looks different than the previous Agora version. @kent @yitongzhang should i customize it to match the look or current version is fine?

Yes plz customize to look like current. Esp important that it hangs to the side so it doenst obscure other votes. thanks!

**Comment by wojtekwtf** [comment]
@yitongzhang i moved it to the left 🤝 

https://www.loom.com/share/6c572d2fc37f47da91c52253afa327ac

**Review by kent — APPROVED** [review]
Comment out the things are slowing it down but overall the love the look and feel

File: src/components/Votes/VoterHoverCard.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    const delegate = await fetchDelegate(addressOrENSName);
+    setLoading(false);
+    setDelegate(delegate);
+  };
+
+  useEffect(() => {
+    fetchDelegateAndSet(address);
+  }, []);
+
+  return (
```
[the remark this answers is not in the document]
Comment (wojtekwtf) [reply src/components/Votes/VoterHoverCard.tsx:70]: Updated version: Now the delegate statement is loaded before the rest of the delegate information

You can see that the voting power and the delegate button still display with a delay, but its more snappy now than before

https://www.loom.com/share/1ca36c1a70f44120905ddc4e9394f9fa?sid=293f96f8-70f9-44b9-9037-95452ed3255d

**Pull request merged**